doc/dev/dev_setup.md

```diff
@@ -20,7 +20,7 @@ or execute the various commands available in the toolbox.
     cd azure-sdk-for-python
     ```
 
-2.  Create a [virtual environment](https://docs.python.org/3/tutorial/venv.html)
+2.  Create a [virtual environment][virtual_environment]
```

Comment:
(deleting + re-adding old comment as a review comment) 
it looks like this this line and the next line should be separated by a newline or period.
Also, I may be hallucinating, but it looks like numbering for steps 3. and 2. are switched on lines 16 and 23.